config/qualifications.yml

```diff
@@ -39,35 +39,59 @@ subtasks:
-  comment to close this issue.
+  as we use the command line every single day for various tasks.
+
+  Work through [Software Carpentry's Unix Shell Course](https://swcarpentry.github.io/shell-novice/) to
+  get the basics down.
+
+  Check in with your lead developer to review the materials and have any questions answered once you are finished.
 ---
 title: Complete a git tutorial
 description: >
-  We use Git in combination with Github to work collaboratively on our code.
-  Log in to O'Reilly's learning tools for free with your UMass account and
-  complete [this](https://learning.oreilly.com/videos/git-essentials-become/9781800565630/)
-  or another approved tutorial and link to it in a comment to close this issue.
+  We use Git as version control and to work collaboratively on our code.
+
+  Work through the relevant sections of the [Pro Git Book](https://git-scm.com/book/en/v2):
+  - [ ] 1.1 - 1.4
+  - [ ] 2.1 - 2.5
+  - [ ] 3.1 - 3.7 (Whole chapter)
+
+  Check in with your lead developer to review the materials and have any questions answered once you are finished.
 ---
 title: Complete a ruby tutorial
 description: >
-  There are many Ruby tutorials available which you can choose from. Download the [Ruby Koans](http://rubykoans.com/),
-  or the free [codeacademy](https://www.codecademy.com/learn/learn-ruby) course.
-  If neither of those options appeal to you you may choose another tutorial with your supervisor's approval.
+  The majority of our applications are written in ruby.
+
+  Work through the [Ruby Koans](http://rubykoans.com/), which are a set of worksheets that teach you ruby
+  syntax/concepts as you go.
+
+  Check in with your lead developer to get set up properly before starting.
+  
+  Check in with your lead developer to review the materials and have any questions answered once you are finished.
+---
+title: Complete a web development crash course
+description: >
+  We write and maintain web applications, so an understanding of web technologies is essential.
+
+  Meet with your lead developer to go over fundamentals and architecture, relational databases, as well as basic
+  HTML/CSS/JS.
 ---
 title: Complete a rails tutorial
 description: >
-  The current tutorial in use is Michael Hartl's [Ruby on Rails Tutorial](https://learning.oreilly.com/library/view/ruby-on-rails/9780136702726/).
-  Log in for free using your UMass account to O'Reilly or, if you wish to use an alternate tutorial, talk to your supervisor.
+  A smaller majority of our applications are written with Ruby on Rails.
+
+  Work through [The Odin Project's Ruby on Rails Course](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails),
+  which will expose you to all the Rails subsystems.
+
+  There are some sections you will need to skip or get alternative explanations/directions about, specifically:
+  - All Projects (i.e. lessons starting with "Project:")
+  - Introduction / Installing PostgreSQL
```

Comment:
I wonder about using a different than "/" character here. You're using it as a path-through-breadcrumbs, but I initially read it as the either-or meaning. Perhaps "→" or somesuch?